hutch_python/_startup_script.py

```diff
@@ -0,0 +1,40 @@
+"""
+Startup script files to be run after ipython is initialized via:
+``c.InteractiveTerminalApp.exec_lines``
+
+Code here will take executed after both IPython has been loaded and
+the namespace has been populated.
+
+safe_load not used to try and keep on-screen messages minimal, as
+they would show up after the IPython banner.
+"""
+
+import logging
+
+import IPython
+from nabs.callbacks import ELogPoster
+from prompt_toolkit.keys import Keys
+
+logger = logging.getLogger(__name__)
+
+# Unbind Ctrl+\\
+try:
```

Comment:
Should these use the `safe_load` context manager to give a description of what it's doing?

Reply:
duplicate, resolving